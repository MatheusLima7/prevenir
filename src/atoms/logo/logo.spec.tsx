import { render } from "@testing-library/react-native";

import Logo from "./logo";

describe("Logo Component", () => {
  it("should render component correctly", () => {
    const container = render(<Logo />);

    expect(container).toMatchSnapshot();
  });
});
