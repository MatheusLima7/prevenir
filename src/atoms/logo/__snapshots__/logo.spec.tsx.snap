// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Component should render component correctly 1`] = `
<Image
  source={
    {
      "testUri": "../../../src/assets/prevenir.png",
    }
  }
  style={
    {
      "height": 83,
      "width": 190,
    }
  }
/>
`;
